src/rgw/rgw_rest_s3.cc

```diff
@@ -2882,7 +2882,7 @@ void RGWDeleteMultiObj_ObjStore_S3::send_partial_response(rgw_obj_key& key,
 
       s->formatter->dump_string("Key", key.name);
       s->formatter->dump_string("VersionId", key.instance);
-      s->formatter->dump_int("Code", r.http_ret);
+      s->formatter->dump_string("Code", r.s3_code);
```

Comment:
Hi Amit,

Sometimes exceeding 50 characters in a PR title (or bz subject) is probably justified.  That's something that the RGW reviewers would probably flag, if it were a problem, in the case of an RGW PR.

cheers,

Matt

Matt